dashboard/config/foorm/library/surveys/pd/ayw_module_1.0.json

```diff
@@ -111,19 +114,19 @@
        ],
        "rows": [
         {
-         "value": "(pre-work)_Reading_and_co",
-         "text": "(pre-work) Reading and comparing the “standard” lesson plan to the “model” lesson plan"
+         "value": "pre_work_reading",
+         "text": "(pre-work) Reading and comparing the \“standard\” lesson plan to the \“model\” lesson plan"
         },
         {
-         "value": "(virtual_meeting)_Checkin",
+         "value": "virtual_meeting_checkin",
          "text": "(virtual meeting) Checking in on how things are going in the classroom. "
         },
         {
-         "value": "(virtual_meeting)_Discuss",
+         "value": "virtual_discuss_teaching_practices",
          "text": "(virtual meeting) Discussing the teaching practices from the model lesson"
         },
         {
-         "value": "(virtual_meeting)_Discuss",
+         "value": "(virtual_discuss_engagement",
```

Comment:
super-nit: do you want a `(` here?

Reply:
not final. And also not what I thought I committed.  hrm